automotive/build.gradle.kts

```diff
@@ -147,6 +147,7 @@ android {
 
 dependencies {
     implementation(project(":common"))
+    implementation(libs.mpgraph)
```

Comment:
Not sure there is a way to use widgets on Automotive, but if you're declaring the dependency then we should also add it to manifest.